javascript/javascript1/week2/formalname.js

```diff
@@ -1,14 +1,14 @@
 let fullname1, fullname2;
 
-function getFullname(firstname, surname, useFormalName = true) {
+function getFullname(firstname, surname, useFormalName) {
```

Comment:
keep the default value for `useFormalName` as `false`, don't remove it.

Reply:
Super!! Thank you!